tests/python/pytorch/graphbolt/test_dataloader.py

```diff
@@ -70,13 +73,22 @@ def test_gpu_sampling_DataLoader(overlap_feature_fetch):
         graph,
         fanouts=[torch.LongTensor([2]) for _ in range(2)],
     )
-    datapipe = dgl.graphbolt.FeatureFetcher(
-        datapipe,
-        feature_store,
-        ["a", "b"],
-    )
+    if enable_feature_fetch:
+        datapipe = dgl.graphbolt.FeatureFetcher(
+            datapipe,
+            feature_store,
+            ["a", "b"],
+        )
 
     dataloader = dgl.graphbolt.DataLoader(
         datapipe, overlap_feature_fetch=overlap_feature_fetch
     )
+    if enable_feature_fetch and overlap_feature_fetch:
```

Comment:
it's better to check if `len(Awaiter)` and `len(Buffer)` equal with expected value: 0 or 1 for all combination of `overlap_xxx` and `enable_xxx`.

Reply:
Got it.